miniapp/proguard-rules.pro

```diff
@@ -28,3 +28,4 @@
 ##---------------End: proguard configuration for Gson  ----------
 
 -keep class com.google.android.gms.ads.MobileAds { *; }
+-keep class com.rakuten.tech.mobile.analytics.Event { *; }
```

Comment:
Do you also need to keep `com.rakuten.tech.mobile.analytics.RatTracker`? I'm not really sure, but it seems like Proguard would fail if the Host App doesn't have the Analytics SDK since it can't find `RatTracker`. Or maybe we need to keep `com.rakuten.tech.mobile.miniapp.analytics.MiniAppAnalytics`?

Reply:
I tested with both mocking and real analytics sdk, no need proguard rule for `RatTracker`.